package.json

```
@@ -255,6 +256,7 @@
 		"lint:css": "stylelint \"**/*.scss\" --syntax scss",
 		"lint:js": "npm run -s install-if-deps-outdated && eslint --ext .js --ext .jsx --ext .ts --ext .tsx --cache .",
 		"lint:mixedindent": "mixedindentlint --ignore-comments \"client/**/*.scss\" \"assets/**/*.scss\" \"**/*.js\" \"**/*.jsx\" \"**/*.tsx\" \"**/*.tsx\" \"!build/**\" \"!node_modules/**\" \"!public/**\" \"!client/config/index.js\"",
+		"lint:package-json": "npmPkgJsonLint './package.json' './packages/**/package.json' './apps/**/package.json'",
```

These globs could likely be "smaller":

```suggestion
		"lint:package-json": "npmPkgJsonLint './package.json' './packages/*/package.json' './apps/*/package.json'",
```

We'd expect to match `packages/foo/package.json`, but not `packages/an/arbitrary/directory/depth/package.json`.